pkg/api/api.go

```diff
@@ -466,7 +466,6 @@ func (hs *HTTPServer) registerRoutes() {
 			})
 
 			dashboardRoute.Post("/calculate-diff", authorize(ac.EvalPermission(dashboards.ActionDashboardsWrite)), routing.Wrap(hs.CalculateDashboardDiff))
-			dashboardRoute.Post("/validate", authorize(ac.EvalPermission(dashboards.ActionDashboardsWrite)), routing.Wrap(hs.ValidateDashboard))
```

Comment:
+1 does not do anything useful now, and is not used.

Reply:
nice, i wasn't sure about that one, but i was hoping it was the case.